test/command/test-base.rb

```diff
@@ -138,9 +138,9 @@ def test_select_command
       def test_brackets_format
         load = Groonga::Command::Base.new("load",
                                           :table => "Site",
+                                          :columns => "_key,title",
                                           :values => <<-VALUES)
   [
```

Comment:
Even though `:columns => _key,title` is specified, I expected that there is no need to remove `["_key", "title"]` from load test data because #to_elasticsearch_format ignores well, but that it seems that it is not true in this context. Is this intended behavior?

Reply:
Oh, I've understood expected Groonga::Command object wrongly in this context.